cluster-autoscaler/main.go

```diff
@@ -280,6 +283,11 @@ func main() {
 	bindFlags(&leaderElection, pflag.CommandLine)
 	flag.Var(&nodeGroupsFlag, "nodes", "sets min,max size and other configuration data for a node group in a format accepted by cloud provider."+
 		"Can be used multiple times. Format: <min>:<max>:<other...>")
+	flag.Var(&nodeGroupAutoDiscoveryFlag, "node-group-auto-discovery", "One or more definition(s) of node group auto-discovery. "+
+		"A definition is expressed `<name of discoverer>:[<key>[=<value>]]`. "+
+		"The `aws` and `gce` cloud providers are currently supported. AWS matches by ASG tags, e.g. `asg:tag=tagKey,anotherTagKey`. "+
+		"GCE matches by IG prefix, and requires you to specify min and max nodes per IG, e.g. `mig:prefix=pfx,min=0,max=10` "+
```

Comment:
Could we make it look like `mig:namePrefix=...` rather than `mig:prefix=...`?
The context is that I've initially made `asg:tag=...` as so because clearly it was about asgs' tag.
On the other hand, for me, `mig:prefix=...`  seems a bit unclear what the prefix is about.
Of course it is the prefix of MIG name but if you'd like to make it even more clearer, I guess it will be good idea to name it something like `namePrefix`. Just my two cents.

Reply:
Good idea - clearer is better.

(Except for all the Java style LongVariableNames in this codebase, which make baby Gophers cry. :trollface:)